documentation/book/snip-images.adoc

```diff
@@ -0,0 +1,32 @@
+[table,stripes=none]
+|===
+|Container image |Namespace/Repository |Description
+
+|Kafka
+a|
+* {DockerOrg}/kafka:{DockerTag}-kafka-2.0.0
+* {DockerOrg}/kafka:{DockerTag}-kafka-2.0.1
+* {DockerOrg}/kafka:{DockerTag}-kafka-2.1.0
+* {DockerOrg}/kafka:{DockerTag}-kafka-2.1.1
+
+a|
+AMQ Streams image for running Kafka, including:
+
+* Kafka broker
+* Kafka Connect
+* Kafka Mirror Maker.
+
+A given image will only be need when running a Kafka broker of that version.
```

Comment:
This is not true because of Zookeeper and Sidecars. Please remove this sentence.

Reply:
Removed